app.js

```diff
@@ -12,7 +15,62 @@ app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+
+// Retrieve an access token
+spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
+
+//Middleware for pasing the json and form-data requests
+app.use(express.json());
+app.use(express.urlencoded({extended: false}));
 
 // Our routes go here:
 
+//HOME PAGE
+app.get('/', (req, res) => {
+    res.render('index');
+  })
+
+
+//ARTISTS RESULTS 
+
+app.get("/artist-search", (req, res) => {
+    const artistName = req.query.artistQuery;
+
+    spotifyApi
+    .searchArtists(artistName, { limit: 5})
+    .then(data => {
+        res.render('artist-search-results', {artistas: data.body.artists.items})
```

Comment:
Just as a suggestion, before passing data to the view, you can save them into a variable to have the code cleaner ;)